files/justfiles/secureblue.just

```diff
@@ -682,3 +695,322 @@ debug-info:
     failed_services=$(echo -e "\n=== Failed Services ===\n"; systemctl list-units --state=failed)
     content="$rpm_ostree_status$sysinfo$flatpaks$audit_results$local_overrides$recent_events$failed_services"
     echo "$content" | fpaste --confirm --private=1
+
+# setup system DNS resolution
+dns-selector:
+    #! /bin/run0 /bin/bash
+    # constants
+    readonly resolved_conf="/etc/systemd/resolved.conf.d/10-securedns.conf"
+    readonly policy_file="/etc/chromium/policies/managed/10-securedns-browser.json"
+    mkdir -p /etc/systemd/resolved.conf.d/
+    mkdir -p /etc/chromium/policies/managed/
+    # variables
+    valid_input="0"
+    resolver_selection=""
+    resolver_subselection=""
+    resolver_has_second_ip=""
+    resolver_supports_ipv6=""
+    resolver_ipv4_address=""
+    resolver_ipv4_address_2=""
+    resolver_ipv6_address=""
+    resolver_ipv6_address_2=""
+    resolver_hostname=""
+    resolver_https_address=""
+    set_browser_policy=""
+
+    echo "Below will be some options to set the DNS resolver for systemd-resolved."
+    echo "All resolvers support DNS-over-TLS (DoT) or DNS-over-QUIC (DoQ), DNS-over-HTTPS (DoH), and DNSSEC."
+    echo "Please select which DNS resolver you would like to set:"
+    echo "    0) Network Resolver - security may vary (system default state)"
+    echo "    1) Control D - has content filtering, anycast"
+    echo "    2) Mullvad - has content filtering, anycast"
+    echo "    3) Cloudflare - very fast with some data collection, anycast"
+    echo "    4) DNSForge - powerful filtering but can be very slow"
+    echo "    5) Custom Resolver - use a custom resolver (must support DoT/DoQ and DNSSEC, ideally also DoH)"
```

Comment:
maybe clarify with "DoH support is also *required* if the browser policy setting is desired*